tasks/EGPSR.tex

```diff
@@ -1,89 +1,76 @@
-\section{Enhanced General Purpose Service Robot [Housekeeper]}
+\section{Enhanced General Purpose Service Robot}
 \label{test:egpsr}
-Similar to a modern smart-speaker, the robot can be asked to do anything, in plain English, from either Stage of this and former rulebooks.
+The robot can be asked to do anything involving abilities required in \SONE{} and \STWO{} of this rulebook.
 
-% \subsection*{Focus}
-% This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
+\noindent \textbf{Focus:} \SysI{}.
 
 \subsection*{Main Goal}
-Execute at least one of the 3 commands requested by the operator.
+Execute 3 commands requested by the operator.
 
-\noindent\textbf{Reward:} 1500pts (500 points per task)\\
+\noindent\textbf{Reward:} 1500pts (500 points per command)
 
-\subsection*{Bonus rewards}
+\subsection*{Bonus Rewards}
 \begin{enumerate}[nosep]
-	\item Attend when called (125pts).
```

Comment:
This is removed bc it's too simple?

Reply:
Mainly, because I don't want to stand in the arena screaming a command for 5 minutes while the robot tries to find me.